v4/source/system.yaml

```diff
@@ -2,18 +2,31 @@
     get:
       tags:
         - system
-      summary: Ping the Server
+      summary: Check system health
       description: |
-        Receive a simple ping response.
+        Check if the system is health or unhealthy based on the configuration setting `GoRoutineHealthThreshold`.
```

Comment:
Can we update this to "Check if the server is up and healthy based on the configuration setting \`GoRoutineHealthThreshold\`. If \`GoRoutineHealthThreshold\` and the number of goroutines on the server exceeds that threshold the server is considered unhealthy. If \`GoRoutineHealthThreshold\` is not set or the number of goroutines is below the threshold the server is considered healthy."